### PR TITLE
Bug fix 

### PR DESCRIPTION
**Pull request by hellodeeplearning**
Added a new py file.

### CODE REVIEW

File: python_file.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import pandas as pd
+
+def hello():
```
Comment (githubtester2022) [inline python_file.py:3]: This does not look correct.

File: python_file.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+	print('hello')
```
Comment (githubtester2022) [inline python_file.py:4]: use a different function

**Review by githubtester2022 — CHANGES_REQUESTED** [review]

File: python_file.py (hunk 2 above)
Comment (hellodeeplearning) [reply python_file.py:4]: Fixed.

File: python_file.py (hunk 1 above)
Comment (hellodeeplearning) [reply python_file.py:3]: Fixed.

**Pull request merged**